frontend/src/TrackFile/FileDetailsConnector.js

```diff
@@ -0,0 +1,77 @@
+import _ from 'lodash';
+import PropTypes from 'prop-types';
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { createSelector } from 'reselect';
+import getErrorMessage from 'Utilities/Object/getErrorMessage';
+import { fetchTrackFiles } from 'Store/Actions/trackFileActions';
+import FileDetails from './FileDetails';
+import LoadingIndicator from 'Components/Loading/LoadingIndicator';
+
+function createMapStateToProps() {
+  return createSelector(
+    (state) => state.trackFiles,
+    (trackFiles) => {
+      return {
+        ...trackFiles
+      };
+    }
+  );
+}
+
+const mapDispatchToProps = {
+  fetchTrackFiles
+};
+
+class FileDetailsConnector extends Component {
+
+  //
+  // Lifecycle
+
+  componentDidMount() {
+    this.props.fetchTrackFiles({ id: this.props.id });
```

Comment:
Do we need a fetch here, shouldn't they already been in the store from the fetch on view load?

Reply:
Yeah, the bulk fetch that happens on mount doesn't read in the metadata tags.  When you ask for a file by specific ID you get a result including the `AudioTag` object